noxfile.py

```diff
@@ -39,18 +39,14 @@ def install_and_run_tests(session, test_dir):
     """Install dependencies and run tests in the specified directory."""
     session.install("setuptools", silent=False)
     session.install("-e", ".[dev]", silent=False)
+    session.run("pipx", "install", ".", "--force", silent=False)
```

Comment:
I don't think it is a good idea to use `pipx` inside a `nox` session, is there an alternate – could we replace this with a standard install? Also, GHA runners come with `pipx` installed, but this might not be the same for local development, so if we were to do this, we would need to install it with `session.install("pipx")`.